include/crypto/zuc.h

```diff
@@ -36,8 +37,14 @@ typedef struct ZUC_KEY_st {
     int inited;
 } ZUC_KEY;
 
+typedef struct eia3_context EIA3_CTX;
+
 void ZUC_init(ZUC_KEY *zk);
 int ZUC_generate_keystream(ZUC_KEY *zk);
 void ZUC_destroy_keystream(ZUC_KEY *zk);
 
+int EIA3_Init(EIA3_CTX *ctx, const unsigned char *key, const unsigned char *iv);
```

Comment:
What is length of key and iv?

Maybe you can refer to Poly1305，something like:

`void Poly1305_Init(POLY1305 *ctx, const unsigned char key[32]);`

Reply:
fixed